task-1-solution/FirstTaskSolution/app/src/main/res/layout/activity_weather.xml

```diff
@@ -7,7 +7,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="?android:attr/textColorSecondary"
+    android:background="?attr/colorSecondary"
```

Comment:
Root layout background rengi için colorSecondary attribute kullanımı Material renk kurallarına göre çoğunlukla uygun değil. Mesela bunun yerine surface renk attribute'u kullanılabilir.

Material 2 ve Material 3 color system dökümantasyonlarına göz atabilirsin.
https://m2.material.io/design/color/the-color-system.html